dom.go

```diff
@@ -750,6 +751,10 @@ func (d document) DocumentURI() string {
 	return d.Get("documentURI").String()
 }
 
+func (d document) ExecCommand(name string, showDefaultUI bool, valueArgument string) bool {
+	return d.Call("execCommand", name, showDefaultUI, valueArgument).Bool()
+}
```

Comment:
I don't expect blank string to be a valid argument, but I haven't formally verified.

If there's already precedent for using `""` when it's not a valid value to indicate something else (`null`), then that should be okay.

However, after briefly looking, it appears blank string may be a valid argument for `insertText`. It's not necessarily a no-op either, because it deletes selection. Also `insertHTML`.

Is the reason to push back against `*string` because it's a hard to use API?

In that case, how about `*ExecCommandArgument`, where `ExecCommandArgument` is a `type ExecCommandArgument struct {Value string}`.